src/components/Nav/UserMenu.vue

```diff
@@ -41,8 +42,14 @@ export default {
     ...mapActions('auth', ['logout']),
     ...mapMutations('user', ['setUserSettings']),
     async wipeClientAndLogout() {
-      await this.logout(true) // Pass true here to propagate this to all clients
-      document.querySelector('.router-view').style.opacity = 0
+      this.loading = true
+
+      try {
+        await this.logout(true) // Pass true here to propagate this to all clients
+        document.querySelector('.router-view').style.opacity = 0
+      } catch {
+        this.loading = false
```

Comment:
do we want this outside of the catch, or are we pushing you off the page on logout so it doesn't matter?

Reply:
We push off the page so we should only need to reset the loading state when there's an error